claasp/components/modsub_component.py

```diff
@@ -39,6 +38,72 @@ def __init__(self, current_round_number, current_round_number_of_components,
         super().__init__(current_round_number, current_round_number_of_components,
                          input_id_links, input_bit_positions, output_bit_size, 'modsub', modulus)
 
+    def algebraic_polynomials(self, model):
+        """
+        Return a list of polynomials representing Modular subtraction operation
+
+        INPUT:
+
+        - ``model`` -- **model object**; a model instance
+
+        EXAMPLES::
+
+            sage: from claasp.cipher_modules.models.algebraic.algebraic_model import AlgebraicModel
+            sage: from claasp.cipher import Cipher
+            sage: cipher = Cipher("cipher_name", "permutation", ["input"], [8], 8)
+            sage: cipher.add_round()
+            sage: modsub_0_0 = cipher.add_MODSUB_component(["input","input"], [[0,1,2,3],[4,5,6,7]], 4)
+            sage: modsub_component = cipher.get_component_from_id('modsub_0_0')
+            sage: algebraic = AlgebraicModel(cipher)
+            sage: modsub_component.algebraic_polynomials(algebraic)
```

Comment:
report the output of the example

Reply:
Done

Reply:
are you sure, I still do not see the output of line 58             
```sage: modsub_component.algebraic_polynomials(algebraic)```

Reply:
it is there